src/Microsoft.OpenApi.OData.Reader/Edm/ODataOperationSegment.cs

```diff
@@ -112,6 +110,25 @@ public override string GetPathItemName(OpenApiConvertSettings settings, HashSet<
             return parameterNamesMapping;
         }
 
+        private string OperationName(IEdmOperation operation, OpenApiConvertSettings settings)
+        {
+            if (settings.EnableUnqualifiedCall)
+            {
+                return operation.Name;
+            }
+            else
+            {
+                if (!string.IsNullOrEmpty(settings.NamespacePrefixToStripForInMethodPaths))
+                {
+                    return operation.Namespace.Replace(settings.NamespacePrefixToStripForInMethodPaths, "").TrimStart('.') + "." + operation.Name;
```

Comment:
replace might have side effects if patterns in the naming repeat, startswith + substring instead to avoid such an issue.